SystemInfo/Collector/DoctrineDatabaseSystemInfoCollector.php

```diff
@@ -39,11 +46,24 @@ public function __construct(Connection $db)
      */
     public function collect()
     {
+        $this->connection = $this->connectionList[$this->siteAccess->name];
```

Comment:
:) That would had been too easy, I'm afraid...
https://doc.ez.no/display/DEVELOPER/SiteAccess#SiteAccess-Limitations includes the following:
Injecting dynamic settings in request listeners is not recommended, as it won't be resolved with the correct scope (request listeners are instantiated before SiteAccess match). Workaround is to inject the ConfigResolver instead, and resolving the setting in your onKernelRequest method (or equivalent).

In constructor the siteaccess is always undefined.